tests/Step/ValidatorStepTest.php

```diff
@@ -10,7 +10,7 @@ class ValidatorStepTest extends \PHPUnit_Framework_TestCase
 {
     protected function setUp()
     {
-        $this->validator = $this->getMockBuilder('Symfony\Component\Validator\Validator')
+        $this->validator = $this->getMockBuilder('Symfony\Component\Validator\Validator\RecursiveValidator')
```

Comment:
Let’s mock `ValidatorInterface` instead. From Symfony 2.5 onwards `validateValue()` become `validate()` so that needs to be changed in the `ValidatorStep`.